index.js

```diff
@@ -308,6 +309,10 @@ DB.prototype._onpt = function (pt, seen, cb) {
   self.log.get(link, function (err, doc) {
     if (doc && doc.value && doc.value.k && doc.value.v) {
       addDoc(doc.value.k, link, doc.value.v)
+    } else if (doc && doc.value && doc.value.d && doc.value.points) {
+      var pt = doc.value.points[0]
+      pt.deleted = true
+      addDoc(doc.value.d, link, pt)
```

Comment:
I'm marking deleted documents as such, but (below) filtering them out. This means no API changes or change in behaviour, but the new logic is in place. This is to make the breaking change PR smaller and a bit easier to reason about.